tsconfig.json

```diff
@@ -10,6 +10,7 @@
     "outDir": "lib",
     "moduleResolution": "node",
     "resolveJsonModule": true,
+    "sourceMap": true,
```

Comment:
We need set to true in for CI and false by default, because it is increase package size